scripts/misc.py

```diff
@@ -2,7 +2,10 @@
 A collection of commonly-used functions, could be cross-projects
 """
 import re
+import logging
```

Comment:
Another file with common functions, although these common functions are specific for import scripts only. Probably it's worth to have this module and do not move all these function to utils, what do you think?

Reply:
I put general functions in misc.py which can be used in different projects while I understand utils.py is for faang backend only.